Adding enable/disable functionality

jb11211 opened the pull request:
By default NTP is now enabled, which improves the previous
behaivour of not starting NTP if it's not running and the
config has not changed

Commit:
Adding enable/disable functionality

By default NTP is now enabled, which improves the previous
behaivour of not starting NTP if it's not running and the
config has not changed

archf commented:
Sorry i kinda mixed state vs 'enablement'. It can be 2 independant vars. 

```yaml
ntpd_state: started
ntpd_enabled: yes

```
And the one task would be something like:

```yaml
- name: start/stop/enable ntpd service
  service:
    name: "{{ ntpd_svc_name }}"
    state: "{{ ntpd_state }}"
    enabled: "{{ ntpd_enabled }}"
  tags: ntp
```

Not tested but should work

Commit:
enable disable seperate variables

Allow ntp config to be set based on whether the
person wants NTP to be started immediately or on reboot

archf commented:
i renamed repo name to fix typo -> now `ansible-ntpd`

jb11211 commented:
Doing the same now.

BTW I�m planning to add Windows support to this thing too

From: archf <notifications@github.com<mailto:notifications@github.com>>
Reply-To: archf/ansible-ntpd <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Sunday, February 28, 2016 at 5:38 PM
To: archf/ansible-ntpd <ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com>>
Cc: "jeb@hp.com<mailto:jeb@hp.com>" <jeb@hp.com<mailto:jeb@hp.com>>
Subject: Re: [ansible-ntpd] Adding enable/disable functionality (#4)


i renamed repo name to fix typo -> now ansible-ntpd

�
Reply to this email directly or view it on GitHub<https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189992340>.

archf commented:
ntpd on windows?

On Sun, Feb 28, 2016 at 8:40 PM, John Brooker <notifications@github.com>
wrote:

> Doing the same now.
>
> BTW I�m planning to add Windows support to this thing too
>
> From: archf <notifications@github.com<mailto:notifications@github.com>>
> Reply-To: archf/ansible-ntpd <reply@reply.github.com<mailto:
> reply@reply.github.com>>
> Date: Sunday, February 28, 2016 at 5:38 PM
> To: archf/ansible-ntpd <ansible-ntpd@noreply.github.com<mailto:
> ansible-ntpd@noreply.github.com>>
> Cc: "jeb@hp.com<mailto:jeb@hp.com>" <jeb@hp.com<mailto:jeb@hp.com>>
> Subject: Re: [ansible-ntpd] Adding enable/disable functionality (#4)
>
>
> i renamed repo name to fix typo -> now ansible-ntpd
>
> �
> Reply to this email directly or view it on GitHub<
> https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189992340>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189992580>.
>



-- 
Felix Archambault
Tel: 514-903-6713
courriel: fel.archambault@gmail.com

jb11211 commented:
No. it will be configuring the native NTP daemon (no point in installing a different one). But from the same role.


From: archf <notifications@github.com<mailto:notifications@github.com>>
Reply-To: archf/ansible-ntpd <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Sunday, February 28, 2016 at 5:53 PM
To: archf/ansible-ntpd <ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com>>
Cc: "jeb@hp.com<mailto:jeb@hp.com>" <jeb@hp.com<mailto:jeb@hp.com>>
Subject: Re: [ansible-ntpd] Adding enable/disable functionality (#4)

ntpd on windows?

On Sun, Feb 28, 2016 at 8:40 PM, John Brooker <notifications@github.com<mailto:notifications@github.com>>
wrote:

> Doing the same now.
>
> BTW I�m planning to add Windows support to this thing too
>
> From: archf <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>>
> Reply-To: archf/ansible-ntpd <reply@reply.github.com<mailto:reply@reply.github.com><mailto:
> reply@reply.github.com<mailto:reply@reply.github.com>>>
> Date: Sunday, February 28, 2016 at 5:38 PM
> To: archf/ansible-ntpd <ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com><mailto:
> ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com>>>
> Cc: "jeb@hp.com<mailto:jeb@hp.com><mailto:jeb@hp.com>" <jeb@hp.com<mailto:jeb@hp.com><mailto:jeb@hp.com>>
> Subject: Re: [ansible-ntpd] Adding enable/disable functionality (#4)
>
>
> i renamed repo name to fix typo -> now ansible-ntpd
>
> �
> Reply to this email directly or view it on GitHub<
> https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189992340>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189992580>.
>



--
Felix Archambault
Tel: 514-903-6713
courriel: fel.archambault@gmail.com<mailto:fel.archambault@gmail.com>

—
Reply to this email directly or view it on GitHub<https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189994054>.

archf commented:
you target windows server or desktop?

On Sun, Feb 28, 2016 at 8:55 PM, John Brooker <notifications@github.com>
wrote:

> No. it will be configuring the native NTP daemon (no point in installing a
> different one). But from the same role.
>
>
> From: archf <notifications@github.com<mailto:notifications@github.com>>
> Reply-To: archf/ansible-ntpd <reply@reply.github.com<mailto:
> reply@reply.github.com>>
> Date: Sunday, February 28, 2016 at 5:53 PM
> To: archf/ansible-ntpd <ansible-ntpd@noreply.github.com<mailto:
> ansible-ntpd@noreply.github.com>>
> Cc: "jeb@hp.com<mailto:jeb@hp.com>" <jeb@hp.com<mailto:jeb@hp.com>>
> Subject: Re: [ansible-ntpd] Adding enable/disable functionality (#4)
>
> ntpd on windows?
>
> On Sun, Feb 28, 2016 at 8:40 PM, John Brooker <notifications@github.com
> <mailto:notifications@github.com>>
> wrote:
>
> > Doing the same now.
> >
> > BTW I�m planning to add Windows support to this thing too
> >
> > From: archf <notifications@github.com<mailto:notifications@github.com
> ><mailto:notifications@github.com>>
> > Reply-To: archf/ansible-ntpd <reply@reply.github.com<mailto:
> reply@reply.github.com><mailto:
> > reply@reply.github.com<mailto:reply@reply.github.com>>>
> > Date: Sunday, February 28, 2016 at 5:38 PM
> > To: archf/ansible-ntpd <ansible-ntpd@noreply.github.com<mailto:
> ansible-ntpd@noreply.github.com><mailto:
> > ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com>>>
>
> > Cc: "jeb@hp.com<mailto:jeb@hp.com><mailto:jeb@hp.com>" <jeb@hp.com
> <mailto:jeb@hp.com><mailto:jeb@hp.com>>
> > Subject: Re: [ansible-ntpd] Adding enable/disable functionality (#4)
> >
> >
> > i renamed repo name to fix typo -> now ansible-ntpd
> >
> > �
> > Reply to this email directly or view it on GitHub<
> > https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189992340>.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189992580>.
> >
>
>
>
> --
> Felix Archambault
> Tel: 514-903-6713
> courriel: fel.archambault@gmail.com<mailto:fel.archambault@gmail.com>
>
> —
> Reply to this email directly or view it on GitHub<
> https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189994054>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189994450>.
>



-- 
Felix Archambault
Tel: 514-903-6713
courriel: fel.archambault@gmail.com

jb11211 commented:
Server. I don’t care about desktop

From: archf <notifications@github.com<mailto:notifications@github.com>>
Reply-To: archf/ansible-ntpd <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Sunday, February 28, 2016 at 6:10 PM
To: archf/ansible-ntpd <ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com>>
Cc: "jeb@hp.com<mailto:jeb@hp.com>" <jeb@hp.com<mailto:jeb@hp.com>>
Subject: Re: [ansible-ntpd] Adding enable/disable functionality (#4)

you target windows server or desktop?

On Sun, Feb 28, 2016 at 8:55 PM, John Brooker <notifications@github.com<mailto:notifications@github.com>>
wrote:

> No. it will be configuring the native NTP daemon (no point in installing a
> different one). But from the same role.
>
>
> From: archf <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>>
> Reply-To: archf/ansible-ntpd <reply@reply.github.com<mailto:reply@reply.github.com><mailto:
> reply@reply.github.com<mailto:reply@reply.github.com>>>
> Date: Sunday, February 28, 2016 at 5:53 PM
> To: archf/ansible-ntpd <ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com><mailto:
> ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com>>>
> Cc: "jeb@hp.com<mailto:jeb@hp.com><mailto:jeb@hp.com>" <jeb@hp.com<mailto:jeb@hp.com><mailto:jeb@hp.com>>
> Subject: Re: [ansible-ntpd] Adding enable/disable functionality (#4)
>
> ntpd on windows?
>
> On Sun, Feb 28, 2016 at 8:40 PM, John Brooker <notifications@github.com<mailto:notifications@github.com>
> <mailto:notifications@github.com>>
> wrote:
>
> > Doing the same now.
> >
> > BTW I�m planning to add Windows support to this thing too
> >
> > From: archf <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com
> ><mailto:notifications@github.com>>
> > Reply-To: archf/ansible-ntpd <reply@reply.github.com<mailto:reply@reply.github.com><mailto:
> reply@reply.github.com<mailto:reply@reply.github.com>><mailto:
> > reply@reply.github.com<mailto:reply@reply.github.com><mailto:reply@reply.github.com>>>
> > Date: Sunday, February 28, 2016 at 5:38 PM
> > To: archf/ansible-ntpd <ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com><mailto:
> ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com>><mailto:
> > ansible-ntpd@noreply.github.com<mailto:ansible-ntpd@noreply.github.com><mailto:ansible-ntpd@noreply.github.com>>>
>
> > Cc: "jeb@hp.com<mailto:jeb@hp.com><mailto:jeb@hp.com><mailto:jeb@hp.com>" <jeb@hp.com<mailto:jeb@hp.com>
> <mailto:jeb@hp.com><mailto:jeb@hp.com>>
> > Subject: Re: [ansible-ntpd] Adding enable/disable functionality (#4)
> >
> >
> > i renamed repo name to fix typo -> now ansible-ntpd
> >
> > �
> > Reply to this email directly or view it on GitHub<
> > https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189992340>.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189992580>.
> >
>
>
>
> --
> Felix Archambault
> Tel: 514-903-6713
> courriel: fel.archambault@gmail.com<mailto:fel.archambault@gmail.com><mailto:fel.archambault@gmail.com>
>
> —
> Reply to this email directly or view it on GitHub<
> https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189994054>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189994450>.
>



--
Felix Archambault
Tel: 514-903-6713
courriel: fel.archambault@gmail.com<mailto:fel.archambault@gmail.com>

—
Reply to this email directly or view it on GitHub<https://github.com/archf/ansible-ntpd/pull/4#issuecomment-189996433>.
